projects/applicationinsights-angularplugin-js/src/lib/applicationininsights-angularplugin-multiple-instace.spec.ts

```diff
@@ -14,7 +14,6 @@ import { Component, Injector } from "@angular/core";
 class FakeHomeComponent {}
 class FakeAboutComponent {}
 describe("ReactAI", () => {
-    let service: ApplicationinsightsAngularpluginErrorService;
```

Comment:
Looks like merge issues?

Reply:
this line is not used in the test file, remove it could reduce the lint error throw by github